### PR TITLE
add like jis(iso) layout keymap for crkbd

### PR DESCRIPTION
**Pull request by marksard**
Hi,
I will addition a keymap for crkbd. Could you please merge it.

Best regards,

### CODE REVIEW

File: keyboards/crkbd/keymaps/like_jis/config.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/crkbd/keymaps/like_jis/config.h]: Could you replace the boiler plate with `#pragma once`? 

Eg, remove this: 
```h
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "../../config.h"
```
And the `#endif` at the end with just `#pragma once`

File: keyboards/crkbd/keymaps/like_jis/keymap.c [hunk 2]
```diff
@@ -0,0 +1,234 @@
+#include "crkbd.h"
```
Comment (drashna) [inline keyboards/crkbd/keymaps/like_jis/keymap.c]: Could you remove (most of) the includes here and replace them with just `#include QMK_KEYBOARD_H`?

File: keyboards/crkbd/keymaps/like_jis/keymap.c [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#include "LUFA/Drivers/Peripheral/TWI.h"
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+#include "../lib/mode_icon_reader.c"
```
Comment (drashna) [inline keyboards/crkbd/keymaps/like_jis/keymap.c]: You should probably call these via the rules.mk file, using `SRC += ...` (replacing the ... with each of the files).  But that is optional here.

File: keyboards/crkbd/keymaps/like_jis/rules.mk [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+ONEHAND_ENABLE = no        # Enable one-hand typing
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/crkbd/keymaps/like_jis/rules.mk]: Could you remove this block?  It's not needed, and is a hold over from the old makefile system

File: keyboards/crkbd/keymaps/like_jis/rules.mk [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+CONSOLE_ENABLE = no         # Console for debug(+400)
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+ONEHAND_ENABLE = no        # Enable one-hand typing
```
Comment (drashna) [inline keyboards/crkbd/keymaps/like_jis/rules.mk]: This should be `SWAP_HANDS_ENABLE`, (it was changed at some point)

File: keyboards/crkbd/keymaps/like_jis/keymap.c (hunk 3 above)
Comment (drashna) [reply keyboards/crkbd/keymaps/like_jis/keymap.c]: Okay, it's probably something more complicated to implement then.   

No worries, then.

**Comment by drashna** [comment]
Thanks!

File: keyboards/crkbd/keymaps/like_jis/keymap.c (hunk 3 above)
Comment (marksard) [reply keyboards/crkbd/keymaps/like_jis/keymap.c]: Thanks

**Pull request merged**